2-standalones/automate.yml

```diff
@@ -1,140 +0,0 @@
-# Configurable environment variables:
```

Comment:
Is there any way we could keep the "regular" docker-compose.yml files?   I believe they have some value both to future developers of this project as well as SalesEng folks who want a "quick" environment that's easy to explain.

I'm okay if they get shuttled off to a different subfolder